README.md

```diff
@@ -1,2 +1,74 @@
 # LatentSpaceBuilder
```

Comment:
Let's split things in two main paragraphs: the human-readable part and the code. Similarly to what we discussed before